content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,183 @@
+further_reading:
+  - link: "/cloud_cost_management/"
+    tag: "Documentation"
+    text: "Learn about Cloud Cost Management"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) automatically allocates the costs of your Google BigQuery resources to individual queries and workloads. Use cost metrics enriched with tags from queries, projects, and reservations to visualize BigQuery workload costs in the context of your entire cloud bill. 
+
+CCM displays costs for resources including query-level analysis, storage, and data transfer on the [**BigQuery dashboard**][1].
+
+## BigQuery pricing models
+
+BigQuery offers multiple pricing components, with CCM focusing on query-related processing costs.
+
+### Query Processing
+
+**On-demand queries**: You pay per query based on the amount of data processed.
+- Costs are directly attributed to individual queries based on bytes processed
+- Includes query-level tags for detailed cost attribution
+
+**Reservation-based queries**: You purchase dedicated processing capacity (slots) in advance at a fixed cost. Multiple queries can share this reserved capacity, making cost attribution more complex but potentially more cost-effective for consistent workloads.
+- Costs of reserved slots are allocated proportionally to queries using those slots
+- Allocation based on slot consumption (`total_slot_ms`) per query
+- Includes idle cost calculation for unused reservation capacity
+
+**Other BigQuery Costs:**
+- **Storage**: Charges for data stored in BigQuery tables (active and long-term storage)
+- **Streaming**: Costs for real-time data ingestion via streaming inserts
+- **Data Transfer**: Charges for moving data between regions or exporting data
+- **BI Engine**: Costs for in-memory analytics acceleration
+- **Other services**: ML training, routine executions, and additional BigQuery features
+
+CCM allocates and enriches costs for both query-processing pricing models, providing detailed cost attribution and tagging for your BigQuery analysis workloads. Learn more about BigQuery services and pricing models [**here**][3].
+
+[**Learn more about optimizing BigQuery performance and costs.**][8]
+
+## Prerequisites
+
+The following table presents the list of collected features and the minimal requirements:
+
+| Feature | Requirements |
+|---|---|
+| Retrieve tags from labels of a query | GCP CCM costs must be setup. Supported without monitoring or reservations. |
+| Query-Level Cost Attribution | BigQuery monitoring enabled |
+| Reservation Cost Allocation | BigQuery reservations configured |
+
+1. Configure the Google Cloud Cost Management integration on the [Cloud Cost Setup page][2].
+2. Enable BigQuery monitoring in your Google Cloud project. 
+[**Enable BigQuery monitoring here**][4]
+3. For reservation cost allocation, configure BigQuery reservations in your project. [**Learn about BigQuery reservations.**][7]
```

Comment:
I dont think we need this one to be honest. It is not really setup. Either they have it enabled and we support allocation or they dont and that is also fine. What do you think?